### DISCUSSION TITLE
consola para  ver javascript

### DISCUSSION THREAD

**Discussion by nataliaramosj** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#425] [General]
hola compañeros , alguien podria explicarme como ver la consola para ver lo que estoy haciendo en javascript ?  adjunto una foto de  que que  que quiero ver  en caso de no haber sido clara con lo que estoy buscando realizar . mi compu es mac . gracias =)
<img width="712" alt="consola" src="https://user-images.githubusercontent.com/99060786/158699522-4b0438fc-89b3-4975-a57c-fb0669248363.png">

**Comment by AngelCruzO** [comment]
Abres tu archivo html en el navegador, aprietas F12 en tu teclado o boton derecho del mouse -> inspeccionar ->consola

**Reply by nataliaramosj** [reply]
ya pude , muchas gracias Angel =)

**Reply by AngelCruzO** [reply]
De nada
